client/src/components/atoms/style/Dropdown.scss

```diff
@@ -0,0 +1,69 @@
+.dropdown{ 
+   .container {
+     max-width: 1200px;
+     width: 100%;
+     margin: auto;
+   } 
+   /* DROPDOWN
+   ============================================ */
+   
+   .dropdown-wrapper {
+     display: flex;
+     min-height: 38px;
+     flex-wrap: wrap;
+     width: 100%;
+     font-family: 'Inter';
+     .dropdown-header {
+        background-color: white;
+        border-color: #ccc;
```

Comment:
The dropdown header doesn't have a border on the designs. Insert this:
```
border: unset;
```

Reply:
Done.